src/vs/workbench/browser/layout.ts

```diff
@@ -102,6 +108,8 @@ export class WorkbenchLayout extends Disposable implements IVerticalSashLayoutPr
 		this.sashXTwo = new Sash(this.workbenchContainer, this);
 		this.sashY = new Sash(this.workbenchContainer, this, { orientation: Orientation.HORIZONTAL });
 
+		this.onMaximizeChange(false);
```

Comment:
@sbatten there is a listener missing to call this when the event happens?

Reply:
its called from workbench.ts@960